torchdynamo/config.py

```diff
@@ -12,14 +13,13 @@
 except ImportError:
     HAS_REFS_PRIMS = False
 
-# print out lots of stuff
-debug = False
-
-# an unreasonable amount of debug printouts
-trace = False
-
-# print the torchdynamo internal exceptions
-print_internal_exceptions = True
+# log level (levels print what it says + all levels listed below it)
+# DEBUG print full traces <-- lowest level + print tracing of every instruction
+# INFO print compiled functions + graphs
+# WARN print warnings (including graph breaks)
+# ERROR print exceptions (and what user code was being processed when it occurred)
+log_level = logging.WARNING
```

Comment:
It may make sense to pull the log level from a `logging.conf` file instead or read it from `argparse`, otherwise if a user has `pip install torchdynamo` when it does get released on pypi they won't be able to easily change the logging level without build from source

Reply:
iiuc, one can easily change the logging from outside, by simply doing 

`torchdynamo.config.log_level = logging.INFO`